leshan-tl-cf-shared-oscore/src/main/java/org/eclipse/leshan/transport/californium/oscore/cf/InMemoryOscoreContextDB.java

```diff
@@ -35,25 +36,32 @@ public class InMemoryOscoreContextDB extends HashMapCtxDB {
 
     private final OscoreStore store;
 
+    public boolean clientRole = false;
```

Comment:
this attribute should probably be final.

Reply:
Yep. Do you prefer to keep it public or shall I make it private?

Reply:
good catch it should be private :+1:

Reply:
Alright! Now done in commit 3aeba3189b4faddd4a741ef044b1422d2040c2be